### ISSUE TITLE
need more examples

### ISSUE THREAD

**Issue by k4shifz** [conformal/gotk3#2]
need more examples,please!!!

**Comment by jrick** [comment]
We're working on implementing more examples.  Is there any particular feature you would like covered?

**Comment by weberc2** [comment]
Could I get an example of a key_press_event? I can't seem to figure this one out. And I'm not particularly familiar with GTK's signal/callback mechanisms.

**Comment by swynter-ladbrokes** [comment]
I'd appreciate an example that shows us how to take the *glib.Object from a callback and get the actual go object from it so I can use one callback for many events.

**Comment by jrick** [comment]
@swynter-ladbrokes that is not possible in the current design.  The types are not polymorphic (interfaces) so a concrete GObject has to be passed there instead, and you lose all type information about the actual type passed in.

**Comment by biij5698** [comment]
yes,we need more examples,please!for example:how to use gtk_css_provider
